projects/vdk-plugins/vdk-structlog/tests/test_structlog.py

```diff
@@ -113,6 +113,34 @@ def test_stock_fields_removal(log_format):
                 assert re.search(stock_field_reps[shown_field], test_log) is not None
 
 
+@pytest.mark.parametrize("log_format", ["console"])
```

Comment:
It's better to have separate test file for each module. 

e.g test_formatter.py test_structlog_plugin.py ... 
 

- This way the test module are more maintainable (no one long file) 
- It's easier to find relevant tests (module X tests are in test_x)

Reply:
I will do this in a separate PR along with the refactoring of the existing code